src/Resources/app/administration/src/component/adyen-refund/adyen-refund.html.twig

```diff
@@ -0,0 +1,47 @@
+{% block adyen_refund %}
+    <sw-card :hero="false" :isLoading="isLoadingTable" :large="false" title="Refunds" v-if="showWidget">
```

Comment:
Could you move the title to the translations file and change to something like `:title="$tc('adyen.refundsTitle')"`

Reply:
Updated in dbc0f94